WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -2986,6 +2986,14 @@ function(){
    this.parent._editDelegateButton();
 }
 
+//overriding no results container getter function for custom behavior
+ZmAccountDelegatesListView.prototype._getNoResultsContainer =
+function() {
+	var div = document.createElement("div");
+	div.className = "ZmListViewTableNoResultsContainer";
```

Comment:
The "no results" container is very generic and is used for all the DwtListView. I have made changes in the DwtListView that allows having custom container with which we can control the stylings as per need. 

Basically introduced  "ZmListViewTableNoResultsContainer" class for no results display where ListView is used inside the table.